addons/mail/static/src/views/web/fields/scheduled_date_field/scheduled_date_dialog.js

```diff
@@ -1,15 +1,16 @@
 import { DateTimeInput } from "@web/core/datetime/datetime_input";
 import { Dialog } from "@web/core/dialog/dialog";
-import { serializeDateTime, today } from "@web/core/l10n/dates";
+import { today } from "@web/core/l10n/dates";
 
 import { Component, useState } from "@odoo/owl";
 
-export class MailComposerScheduleDialog extends Component {
-    static template = "mail.MailComposerScheduleDialog";
+export class ScheduledDateDialog extends Component {
+    static template = "mail.ScheduledDateDialog";
     static props = {
         close: Function,
-        isNote: Boolean,
-        schedule: Function,
+        isRemovable: { type: Boolean },
+        save: Function,
+        scheduledDate: { type: [luxon.DateTime, Boolean] },
```

Comment:
I'd have used an optional props, but I guess it's easier to handle a record fields value like this ;p